src/crc32c.h

```diff
@@ -26,10 +26,13 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#pragma once
+#ifndef _CRC32C_H_
```

Comment:
prefix non rd-prefixed files with `_RD_<FILENAME>_H_` to avoid collision with system / 3rd-party headers.